projects/vdk-plugins/vdk-impala/tests/functional/test_ingest_to_impala.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import os
+import pathlib
+from unittest import TestCase
+from unittest.mock import patch
+
+import pytest
+from vdk.plugin.impala import impala_plugin
+from vdk.plugin.test_utils.util_funcs import cli_assert_equal
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import get_test_job_path
+
+VDK_DB_DEFAULT_TYPE = "VDK_DB_DEFAULT_TYPE"
+VDK_IMPALA_HOST = "VDK_IMPALA_HOST"
+VDK_IMPALA_PORT = "VDK_IMPALA_PORT"
+VDK_INGEST_METHOD_DEFAULT = "VDK_INGEST_METHOD_DEFAULT"
+
+
+@pytest.mark.usefixtures("impala_service")
+@patch.dict(
+    os.environ,
+    {
+        VDK_DB_DEFAULT_TYPE: "IMPALA",
+        VDK_IMPALA_HOST: "localhost",
+        VDK_IMPALA_PORT: "21050",
+        VDK_INGEST_METHOD_DEFAULT: "IMPALA",
+    },
+)
+class IngestToImpalaTests(TestCase):
+    def test_ingest_to_impala(self):
+        runner = CliEntryBasedTestRunner(impala_plugin)
+
+        runner.invoke(
+            [
+                "impala-query",
+                "--query",
+                "CREATE TABLE test_table (some_data varchar, more_data varchar)",
+            ]
+        )
+
+        ingest_job_result = runner.invoke(
+            [
+                "run",
+                get_test_job_path(
+                    pathlib.Path(os.path.dirname(os.path.abspath(__file__))),
+                    "ingest-job",
```

Comment:
Maybe some files are missing, as I am not able to find the said "ingest-job"?

Reply:
You did find it as one of your comments is aimed at the file in it - https://github.com/vmware/versatile-data-kit/pull/643#discussion_r772161629

Reply:
(facepalm) Correct. Somehow I missed the directory name. :)